arangod/RocksDBEngine/RocksDBReplicationContext.cpp

```diff
@@ -103,6 +134,22 @@ RocksDBReplicationContext::RocksDBReplicationContext(RocksDBEngine& engine, doub
 RocksDBReplicationContext::~RocksDBReplicationContext() {
   MUTEX_LOCKER(guard, _contextLock);
   _iterators.clear();
+
+  try {
+    size_t removed = 0;
+    for (auto& it : _blockers) {
+      for (auto& it2 : it.second) {
+        findCollection(it.first, it2.first, [&](TRI_vocbase_t& vocbase, LogicalCollection& collection) {
+          auto* rcoll = static_cast<RocksDBMetaCollection*>(collection.getPhysical());
+          rcoll->removeRevisionTreeBlocker(TransactionId(it2.second));
+          ++removed;
+        });
+      }
+    }
+  } catch (...) {
```

Comment:
Actually do we want to move the try catch  deeper inside?
If only one blocker could not be removed we will not try the other ones, and leave even more behind.

Also should we add a LOG statement here?
Just in case this happens we can check in the Logs which will then state "FAILED TO remove Blocker <seqid> on <dbname>/<collection>"

Afaik blockers are garbage collected.
So we can add another log on garbage collection.
=> If one is missing we still have blockers preventing stuff. (In this case INFO is good enough)

This should not happen often anyways, but will give us more evidence.

Reply:
Addressed.